tests/virt/cluster/migration_and_maintenance/rbac_hardening/test_migration_rights.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+from kubernetes.dynamic.exceptions import ForbiddenError
+from ocp_resources.cluster_role import ClusterRole
+from ocp_resources.role_binding import RoleBinding
+
+from utilities.constants import UNPRIVILEGED_USER
+from utilities.virt import VirtualMachineForTests, fedora_vm_body, migrate_vm_and_verify, running_vm
+
+pytestmark = pytest.mark.rwx_default_storage
+
+
+@pytest.fixture(scope="session")
+def kubevirt_migrate_cluster_role(admin_client):
```

Comment:
Is this going to be used in the future?
if the only use case of this is for the `kind` and `name` I think you can directly insert them without the need for this fixture.

Reply:
i think it's better to keep 2 separate fixtures:
https://github.com/RedHatQE/openshift-virtualization-tests/blob/main/docs/CODING_AND_STYLE_GUIDE.md?plain=1#L76

Reply:
good point. added